registry/config.go

```diff
@@ -302,34 +315,22 @@ func splitReposName(reposName string) (string, string) {
 
 // NewRepositoryInfo validates and breaks down a repository name into a RepositoryInfo
 func (config *ServiceConfig) NewRepositoryInfo(reposName string, bySearch bool) (*RepositoryInfo, error) {
```

Comment:
Not here. I rather minimize the changes we need for the patch release. That's not necessary to fix the bug.